apps/sql_data/tests/test_admin.py

```diff
@@ -13,12 +13,12 @@
 fake = Faker()
 
 
-class InitializeTestData(LoggedInMixin, APITestCase):
+class TestCoreModels(LoggedInMixin, APITestCase):
```

Comment:
This name is misleading or this test case is in the wrong module. Use a more appropriate name.